app/views/components/tariff_body_partial.scala.html

```diff
@@ -27,7 +27,7 @@
     }
     @if(m3.nonEmpty){
     <li>
-        @m2
+        @m3
```

Comment:
Probably it will be more readable if this `m3` will have better name :D

Reply:
It's almost like varargs or indeed lists were a new concept.